lib/graphql/interface_type.rb

```diff
@@ -23,4 +23,14 @@ def kind
   def possible_types
     @possible_types ||= []
   end
+
+  # @return [GraphQL::Field] The defined field for `field_name`
+  def get_field(field_name)
+    self.fields[field_name]
```

Comment:
nit: lose the `self` here and in `all_fields` to be consistent with other places ?